packages/jsii-pacmak/test/expected.jsii-calc/python/src/jsii_calc/__init__.py

```diff
@@ -31,17 +31,18 @@
 import builtins
```

Comment:
I expected 0 changes in this file as proof that this is not going to break anyone... Can we avoid?

Reply:
Could, but it breaks the logical grouping of imports and it's a little sucky. This is literally just a line moving a couple lines higher... I don't see how this is going to break anything or anyone...

Reply:
I am fine with leaving this as is, but in the future, please resist the urge to introduce unrelated changes in PRs. As much as I understand the desire to make small improvements like this, by definition it introduces risk to the project because it makes the PR harder to review and harder to reason about the implications of the actual change. If you want to reorganize the imports, you can easily just submit a PR that includes only this change and it will be easy to approve and merge...